notebooks/quantum-machine-learning/introduction.ipynb

```diff
@@ -170,17 +174,19 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "There are two distinct categories of **QC** algorithms: those that require quantum random access memory ([qRAM](gloss:qram)), where data can be accessed in superposition, and those that don't. The various proposed qRAM-based QML algorithms, e.g. [qPCA](https://doi.org/10.1038/nphys3029), [qSVM](https://doi.org/10.1103/PhysRevLett.113.130503) and [qClustering](https://arxiv.org/abs/1307.0411), boast exponential speedups compared to their classical algorithms, however there are currently no viable hardware candidates for realizing qRAM. \n",
+    "There are two distinct categories of **QC** algorithms: those that require quantum Random Access Memory ([qRAM](gloss:qram)), where data can be accessed in superposition, and those that don't. The various proposed qRAM-based QML algorithms, e.g. [qPCA](https://doi.org/10.1038/nphys3029), [qSVM](https://doi.org/10.1103/PhysRevLett.113.130503) and [qClustering](https://arxiv.org/abs/1307.0411), boast exponential speedups compared to their classical algorithms, however there are currently no viable hardware candidates for realizing qRAM. \n",
```

Comment:
```suggestion
    "There are two distinct categories of **QC** algorithms: those that require quantum random access memory ([qRAM](gloss:qram)), where data can be accessed in superposition, and those that don't. The various proposed qRAM-based QML algorithms, e.g. [qPCA](https://doi.org/10.1038/nphys3029), [qSVM](https://doi.org/10.1103/PhysRevLett.113.130503) and [qClustering](https://arxiv.org/abs/1307.0411), boast exponential speedups compared to their classical algorithms, however there are currently no viable hardware candidates for realizing qRAM. \n",
```

This should stay as lower-case as it's not a proper noun (for example, see https://en.wikipedia.org/wiki/Random-access_memory).